### ISSUE TITLE
absolute position in .progressCircle, necessary? 

### ISSUE THREAD

**Issue by avivbiton** [mesthabro/react-gradient-progress#3]
I am no expert when it comes to CSS, so please test this yourself before you decide.  
Is this CSS even necessary? 
`.progressCircle {
  position: absolute;
}`
I find the position absolute causing me some problems (but again it may be my project's CSS that interfere here) 
The component works fine by itself, but when I try to add it into other components it causes some issues.
For example, I try to add this component inside a Bootstrap button, and it looks like this:  
![without](https://user-images.githubusercontent.com/39068976/63935117-85fd7a00-ca65-11e9-9836-7d22b63c311b.PNG)

After removing the property `position: absolute`  from .progressCircle
![with](https://user-images.githubusercontent.com/39068976/63935205-aa595680-ca65-11e9-99b0-3414b1051252.PNG)  
  
Now it looks better and the text aligns correctly with the circle

It seems like the component still works without this property.  
Even moving this outside the button it still looks the same and as well takes much less space.

The Component outside the button with `position: absolute`
![withoutButton](https://user-images.githubusercontent.com/39068976/63935720-beea1e80-ca66-11e9-92f6-b6c19d0485a2.png)

And without `position: absolute`
![withButton](https://user-images.githubusercontent.com/39068976/63935795-e50fbe80-ca66-11e9-9043-9ac7120755b2.png)
(the circle takes less space on the screen and again aligns better)
  
TLDR: 
Remove `position: absolute` from `.progressCircle` Good or not?

**Comment by prasannamestha** [comment]
`position: absolute` is required for keeping the progressbar aligned properly. On the other hand, the outer component's positioning is set to `relative` so that you can adapt it as per your liking. Or maybe, I could think of some other way to position this so that it does not interfere with outside css

**Comment by avivbiton** [comment]
It is required for everything but the `.progressCircle` class.  
Removing the absolute position from  `.reactGradientProgressPercentage` or `.reactGradientProgressPercentageSpan` will break the layout of the circle, but that is not the case for `.progressCircle`

**Comment by avivbiton** [comment]
I will test this soon enough with an empty React Project with no CSS, just to check it's not my CSS that is interfering with the component. I also run Bootstrap in my project.

**Comment by avivbiton** [comment]
Update: I find that setting the fontSize to `inherit` will fix any alignment issues caused by removing the `position: absolute` from `.progressCircle`

**Comment by avivbiton** [comment]
#4 Opened a pull request for now with the changes. Need testing.

**Comment by prasannamestha** [comment]
Thanks for the contribution @avivbiton. I have merged your changes to the repo :)
